tools/test-case-generators/format-request-map.json

```diff
@@ -10,6 +10,19 @@
       "repositories": [],
       "filename": "image.vhdx",
       "blueprint": {}
+    },
+    "rhel-8": {
```

Comment:
Maybe make this an explicit dict, called `overrides` or `distros` or something like that, and make the distros entries of that? Just to namespace it correctly.

Reply:
oki